lib/phoenix_live_reload/live_reloader.ex

```diff
@@ -119,17 +119,16 @@ defmodule Phoenix.LiveReloader do
 
   defp before_send_inject_reloader(conn, endpoint) do
     register_before_send(conn, fn conn ->
-      if conn.resp_body != nil && html?(conn) do
-        resp_body = IO.iodata_to_binary(conn.resp_body)
-        if has_body?(resp_body) and :code.is_loaded(endpoint) do
-          [page | rest] = String.split(resp_body, "</body>")
-          body = page <> reload_assets_tag(conn, endpoint) <> Enum.join(["</body>" | rest], "")
-          put_in conn.resp_body, body
-        else
-          conn
-        end
+      with %{resp_body: resp_body} when not is_nil(resp_body) <- conn,
+           true <- html?(conn),
+           true <- has_body?(resp_body),
+           {:file, _} <- :code.is_loaded(endpoint),
+           [leading, trailing] <- :string.split(resp_body, "</body>")
```

Comment:
```suggestion
           [leading | trailing] <- :string.split(resp_body, "</body>")
```

Reply:
The problems is that `:binary` will not work there in case of `:binary.split([255, "</bo", ?d, 'y', [">"], 255], "</body>")` while `:string.split([255, "</bo", ?d, 'y', [">"], 255], "</body>")` will work as expected.

Reply:
Doh, I see. Sorry for butchering your PR and missing the point. I will revert this, we can merge it but probably in a very far away future.

Reply:
Question is whether we want to keep such backward compatibility. Phoenix 1.5 is Elixir 1.9+ only and it uses OTP 20+ which according to docs should have all functions used there.

Reply:
Phoenix v1.5 is for new apps. Phoenix itself supports back to 1.5 or 1.6.